services/QuillLMS/engines/evidence/lib/evidence/evidence/synthetic/seed_data_generator.rb

```diff
@@ -162,6 +163,15 @@ def run
         conjunction_exclusions.any?{|regex| regex.match(text.strip) }
       end
 
+      # Use a struct since Opinion API calls .text on the prompt
+      MockPrompt = Struct.new(:text, keyword_init: true)
```

Comment:
This struct feels a little awkward to me, but I think it makes sense to do it this way until we've integrated all of the various pre-filter APIs we want to run and then see if it makes sense to refactor this only when we've seen what other patterns we might need to support.

Reply:
Yeah, that's fair. I'll use `::Evidence::Prompt.new` instead to match the type expected by the method (rather than mocking the type). Thanks for the feedback.